lite/valor_lite/classification/annotation.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
```

Comment:
would it make sense to rename this file to something like "classification_schemas"? "annotation" seems like an odd choice

Reply:
My reasoning is as follows:

a classification is a form of annotation that applies labels and scores to a datum.

a detection is a form of annotation that applies labeled + scored geometries to a datum